### ISSUE TITLE
autoupdate function fails when calling use_prcommit with custom path_root

### ISSUE THREAD

**Issue by krzyslom** [lorenzwalthert/precommit#92]
Hi,

I'm using precommit v0.0.0.9017 (tag v0.0.0.9027) and I've created the following function inside my package

```{r}
start_project <- function(name) {
  dir.create(name)
  with_dir(name, {
    use_git()
    use_precommit(open = FALSE)
  })
}
```

`start_project("my_proj")` call results in the following error

>An error has occurred: FatalError: git failed. Is it installed, and are you in a Git repository directory?
Check the log at /home/rstudio/.cache/pre-commit/pre-commit.log
Error: Running precommit autoupdate failed.
Run `rlang::last_error()` to see where the error occurred.

Changing the use_precommit call to `use_precommit(open = FALSE, path_root = getwd())` results in the following error

>Error in file(file, "rt", encoding = fileEncoding) : 
  cannot open the connection
In addition: Warning message:
In file(file, "rt", encoding = fileEncoding) :
  cannot open file '.pre-commit-config.yaml': No such file or directory

This is caused by the fact that the use_precommit function doesn't pass the path_root parameter to the autoupdate function - https://github.com/lorenzwalthert/precommit/blob/v0.0.0.9027/R/setup.R#L25-L44
Simply changing `autoupdate()` to `autoupdate(path_root)` fixes the problem.

Please let me know if you'd like me to provide a PR.

**Comment by lorenzwalthert** [comment]
uups, yes. Thanks. Welcome to make a PR if you want. I agree it should be passed.

**Comment by lorenzwalthert** [comment]
I think this is closed with #93.
